benchmark/misc/next-tick-breadth-args.js

```diff
@@ -0,0 +1,21 @@
+
+var common = require('../common.js');
+var bench = common.createBenchmark(main, {
+  millions: [2]
+});
+
+function main(conf) {
+  var N = +conf.millions * 1e6;
+  var n = 0;
+
+  function cb(arg1, arg2, arg3) {
+    n++;
+    if (n === N)
+      bench.end(n / 1e6);
+  }
+
+  bench.start();
+  for (var i = 0; i < N; i++) {
+    process.nextTick(cb, 512, true, null);
```

Comment:
To get more realistic results out of these benchmarks,  we should be calling `process.nextTick` with a rotating number of arguments instead of a static number.